### ISSUE TITLE
[Bug] Fidget text messed up by the buffer behind it.

### ISSUE THREAD

**Issue by luiz00martins** [j-hui/fidget.nvim#84]
This bug has two parts:
 1. When fidget is above some highlighted text in the buffer, it'll have that text's highlight.
 2. If there's a blank space on fidget's text, and a character in the buffer underneath it, that character will override the blank space.

## Example.

This is some text in the buffer that has some syntax highlight:
![Screenshot from 2022-05-30 19-40-14](https://user-images.githubusercontent.com/43142209/171288506-5675d8ae-9810-4a7b-9eba-f383d3dd0da7.png)

This is what happens when fidget tries to display the text `num-traits [cargo check]` above it:
![Screenshot from 2022-05-30 19-40-08](https://user-images.githubusercontent.com/43142209/171288674-6836b34e-5078-4633-b982-50ff1f34da87.png)

The two blank spaces are replaced by the `.` and `r` in the buffer behind them, leading to `num-traits.[cargorcheck]`, and the syntax highlighted is messed up by "inheriting" from the text underneath.

**Comment by syphar** [comment]
[Fidget uses two hilight groups](https://github.com/j-hui/fidget.nvim/blob/main/doc/fidget.md#highlights), most likely you set either `FidgetTask` to a transparent background, or the theme you're using did the same for the `LineNr` hl-group

**Comment by luiz00martins** [comment]
No luck. Same problem with the following config:
```lua
require("fidget").setup{
	window = {
		blend = 0,
	},
}
vim.cmd[[highlight FidgetTitle ctermfg=0 guifg=#000000]]
vim.cmd[[highlight FidgetTask ctermfg=0 guifg=#000000]]
```

**Comment by syphar** [comment]
Perhaps it works when you set the background, not the foreground? :)

**Comment by syphar** [comment]
Then I'm out of ideas, sorry. I had the same issue and for me it worked after setting winblend. 

And I don't know enough about hilight setting / group :)

**Comment by j-hui** [comment]
Hmm this is super weird. I don't have any ideas as to what might cause this either ):

What version of nvim are you using?

**Comment by j-hui** [comment]
@luiz00martins are you using Packer to manage your plugins by any chance? If so, you may need to `PackerCompile`, because the `config` hook you specify is pre-compiled and cached.

**Comment by luiz00martins** [comment]
Makes sense. Yes, I'm using Packer, but after the first manual run, the config worked fine on all subsequent runs.

Thanks for the tip though, I'll keep that in mind the next time I configure it.
